src/main/java/reposense/authorship/FileInfoAnalyzer.java

```diff
@@ -160,14 +166,15 @@ private void aggregateBlameAuthorModifiedAndDateInfo(RepoConfiguration config, F
             String authorName = blameResultLines[lineCount + 1].substring(AUTHOR_NAME_OFFSET);
             String authorEmail = blameResultLines[lineCount + 2]
                     .substring(AUTHOR_EMAIL_OFFSET).replaceAll("<|>", "");
-            Long commitDateInMs = Long.parseLong(blameResultLines[lineCount + 3].substring(AUTHOR_TIME_OFFSET)) * 1000;
+            long commitDateInMs = Long.parseLong(blameResultLines[lineCount + 3].substring(AUTHOR_TIME_OFFSET)) * 1000;
             LocalDateTime commitDate = LocalDateTime.ofInstant(Instant.ofEpochMilli(commitDateInMs),
                     config.getZoneId());
             Author author = config.getAuthor(authorName, authorEmail);
 
-            if (!fileInfo.isFileLineTracked(lineCount / 5) || author.isIgnoringFile(filePath)
+            int lineNumber = lineCount / 5;
```

Comment:
Sure, but perhaps in another PR? The changes is with regrads to existing codes which is beyond the scope of this PR. A quick check on this, we may need to update `getGitBlameResult` and `getGitBlameWithPreviousAuthorsResult` as well to return a list of `GitBlameLineInfo`s. The current `GitBlameLineInfo` also need some update to add in new params.

Reply:
> changes is with regrads to existing codes which is beyond the scope of this PR

It seems like the git blame functions are only really being used in this function (through `getGitBlameResult` and `getGitBlameWithPreviousAuthorsResult`) which we're refactoring anyway - so I think it's fine to make this change in this PR.

I am fine with doing this in another PR too since this would cascade into some test updates.